__tests__/app.test.js

```diff
@@ -13,3 +26,22 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topics, each with slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+       console.log(topics)
+        expect(topics).toHaveLength(3)
```

Comment:
Well done for asserting against the length here to ensure the test does not pass by default if we get an empty array back! :D